crates/store/re_chunk/src/transport.rs

```diff
@@ -396,42 +396,65 @@ impl Chunk {
         {
             re_tracing::profile_scope!("timelines");
 
-            for (timeline, info) in timelines {
-                let TimeColumn {
-                    timeline: _,
-                    times,
-                    is_sorted,
-                    time_range: _,
-                } = info;
-
-                let field = ArrowField::new(
-                    timeline.name().to_string(),
-                    times.data_type().clone(),
-                    false, // timelines within a single chunk are always dense
-                )
-                .with_metadata({
-                    let mut metadata = TransportChunk::field_metadata_time_column();
-                    if *is_sorted {
-                        metadata.extend(TransportChunk::field_metadata_is_sorted());
-                    }
-                    metadata
-                });
+            let mut timelines = timelines
+                .iter()
+                .map(|(timeline, info)| {
+                    let TimeColumn {
+                        timeline: _,
+                        times,
+                        is_sorted,
+                        time_range: _,
+                    } = info;
+
+                    let field = ArrowField::new(
+                        timeline.name().to_string(),
+                        times.data_type().clone(),
+                        false, // timelines within a single chunk are always dense
+                    )
+                    .with_metadata({
+                        let mut metadata = TransportChunk::field_metadata_time_column();
+                        if *is_sorted {
+                            metadata.extend(TransportChunk::field_metadata_is_sorted());
+                        }
+                        metadata
+                    });
+
+                    let times = times.clone().boxed() /* cheap */;
+
+                    (field, times)
+                })
+                .collect_vec();
 
+            timelines.sort_by(|(field1, _times1), (field2, _times2)| field1.name.cmp(&field2.name));
```

Comment:
a bit surprised we care about the order in the transport chunk?

Reply:
We do! RecordBatches respect the order of columns so keeping it stable is pretty important, otherwise the UX for end users is pretty awful.